CMakeLists.txt

```diff
@@ -1,111 +1,104 @@
-cmake_minimum_required(VERSION 3.14)
+cmake_minimum_required(VERSION 3.14...3.20)
 
 project(
-    example
-    VERSION 0.1.0
-    LANGUAGES CXX
+  example
+  VERSION 0.1.0
+  LANGUAGES CXX
 )
 
+option(BUILD_SHARED_LIBS "default" YES)
```

Comment:
`BUILD_SHARED_LIBS` must **never** be set in project CMLs.

Reply:
What do you mean with CML?
Were is this written?

Reply:
**CM**ake**L**ists.txt